finalPrep/annotations.js

```diff
@@ -2,10 +2,15 @@
 // Use the double // to create a new comment
 
 //Build a Bear
+  //this is a function that with parameters name, age, fur, clothes, and specialPower
 function buildABear(name, age, fur, clothes, specialPower) {
+  //declare a variable called greeting, value is a string that interpolates the name parameter
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+  //declare a variable called demographic, variable is an array with 2 elements: name and age
   var demographics = [name, age];
+  //declare variable called powerSaying, value is a string that concatenates with the specialPower parameter
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+  //declare a variable called 'builtBear'. The value is an Object that contains 6 key-value pairs
```

Comment:
It can still be worth going line by line through this object data- where do all the values come from? parameters? Previously declared variables?